#this is the main message for map and neighbor updates in hare

uint8 robot_id

<<<<<<< Updated upstream
cell[] cellUpdate
=======
# should this be int4
hare/Obstacle[] sensing
>>>>>>> Stashed changes

nav_msgs/Odometry robot_odom

std_msgs/String tree_state
